### PR TITLE
Add NFV service

### PR DESCRIPTION
**Pull request by Ladas**
Add NFV service requests and models. Adding also request and
models tests and documentation.

### CODE REVIEW

**Comment by Ladas** [comment]
damn, hound just got crazy and it stopped using our rubocop.yml

File: lib/fog/openstack/models/collection.rb [hunk 1]
```diff
@@ -34,7 +34,10 @@ def summary(options = {})
       def get(uuid)
         raise Fog::OpenStack::Errors::InterfaceNotImplemented.new('Method :get is not implemented')
       end
-      alias_method :find_by_id, :get
+
+      def find_by_id(uuid)
+        get(uuid)
+      end
```
Comment (dhague) [inline lib/fog/openstack/models/collection.rb]: What does this achieve that alias_method didn't?

Comment (Ladas) [inline lib/fog/openstack/models/collection.rb]: gah, seems like this bug was here quite a while. Alias works weird with inheritance, so it always points to the above method, not the one implemented in the subclass. The things we discover with model tests. :-)

Comment (dhague) [inline lib/fog/openstack/models/collection.rb]: In which case: good catch!

**Pull request merged**